linalg/sparsemat.hpp

```diff
@@ -118,6 +145,12 @@ class SparseMatrix : public AbstractSparseMatrix
    /// Create a SparseMatrix with diagonal @a v, i.e. A = Diag(v)
    SparseMatrix(const Vector & v);
 
+   // Initialize CuSparse
+   void InitCuSparse();
```

Comment:
I don't think this method needs to be `public`.

Reply:
Good point! Moved to protected 👍.